packages/editor/src/components/KeyboardEventWrapper.tsx

```diff
@@ -32,6 +32,10 @@ export const KeyboardEventWrapper: React.FC<Props> = ({
     switch (e.key) {
       case 'Delete':
       case 'Backspace':
+        // if `selectionStart` available,think it is textarea or text input
+        if ((e.target as any).selectionStart !== undefined) {
```

Comment:
looks like checking the target's tag name is a more straightforward and easy-to-understand way of doing this?

Reply:
seems better，i'll check it out

Reply:
Checking tag name cannot detect whether the input is focused.

Reply:
> Checking tag name cannot detect whether the input is focused.

It should be possible, the result of my test is that the tagname is `INPUT` only in the focus state